minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -113,8 +116,15 @@
 
   private MultiProcessCluster(Map<PropertyKey, String> properties,
       Map<Integer, Map<PropertyKey, String>> masterProperties,
-      Map<Integer, Map<PropertyKey, String>> workerProperties,
-      int numMasters, int numWorkers, String clusterName, DeployMode mode) {
+      Map<Integer, Map<PropertyKey, String>> workerProperties, int numMasters, int numWorkers,
+      String clusterName, DeployMode mode, List<PortCoordination.ReservedPort> ports) {
+    if (System.getenv(ALLUXIO_USE_FIXED_TEST_PORTS) != null) {
+      Preconditions.checkState(
+          ports.size() == numMasters * PORTS_PER_MASTER + numWorkers * PORTS_PER_WORKER,
+          "Require 2 ports per master and 3 ports per worker, but there are %s masters, "
```

Comment:
NIT: this message should use `PORTS_PER_MASTER` and `PORTS_PER_WORKER`.

Reply:
done, thanks